src/grt/test/silence.ok

```diff
@@ -9,4 +9,6 @@
 [INFO ODB-0131]     Created 676 components and 2850 component-terminals.
 [INFO ODB-0133]     Created 579 nets and 1498 connections.
 [INFO ODB-0134] Finished DEF file: gcd.def
-No differences found.
+Differences found at line 642.
```

Comment:
You missed only this `guideok`. Update this and the grt tests will be all correctly updated.